printer_mods/See1807/Voron V0 Exhaust and LED Mod/README.md

```diff
@@ -0,0 +1,38 @@
+# Voron V0 Exhaust and LED Mod
+## _No stink, good light_
+
```

Comment:
Please add the images you included to this README so they're visible without extra clicks.